<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget_Documentos">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Datos">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Registro de documento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AnyadirDoc">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Añadir documento</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Sexo">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>M / H</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mujer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hombre</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellidos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Número:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Nacionalidad">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Nombre">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NumDoc">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Apellidos">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nacionalidad:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_TipoDoc">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DNI / NIE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DNI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NIE</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de documento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexo:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
